src/marked.js

```diff
@@ -3,6 +3,31 @@ import matchAll from 'match-all';
 import { customHeadingRenderer } from './markdown_renderers';
 
 
+/**
+ * Map from the language of a fenced code block to the title of corresponding tab.
+ * The language is a string provided by the default marked tokenizer.
+ * Note that not all of the languages (such as python2) might be possible at the moment
+ * in the default marked tokenizer. We anyway include them here for
+ * robustness to potential future improvements of marked.
+ * In case tab title can't be resolved from language using this mapping, the language itself is used as a tab title.
+ */
+const LANGAUGE_TO_TAB_TITLE = {
+    js: 'Node.JS',
+    javascript: 'Node.JS',
+    nodejs: 'Node.JS',
+    bash: 'Bash',
+    curl: 'cURL',
+    dockerfile: 'Dockerfile',
+    php: 'PHP',
+    json: 'JSON',
+    xml: 'XML',
+    python: 'Python',
+    python2: 'Python3',
+    python3: 'Python3',
+    yml: 'YAML',
```

Comment:
The first 'Python3' -> 'Python 2'
The second 'Python3' -> 'Python 3' (with a space), pls